### PR TITLE
First version of prod

### PR DESCRIPTION
**Pull request by TheKnarf**
Prodsetting:

1. Ensure that terraform is up to date:
    
```
cd packages/infrastructure
terraform plan
# terraform apply # only if needed
```

2. Login to azure (needs `az`-, `jq`- and `docker`-command installed):

```
make az-docker-login
```

3. Build and deploy docker containers:

```
make build-website build-cms
```

4. Run docker compose up:

```
make docker-compose-up
```

You may need to create the docker context `myacicontext`, forgot what command I ran to do that. I think maybe it was created when I first tried to login?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+az-docker-login: ## logs you in to azure and uses those credentials for docker
+	az acr login --name dockercontainerregistry2020 --expose-token|jq .accessToken -r|docker login dockercontainerregistry2020.azurecr.io --username 00000000-0000-0000-0000-000000000000 --password-stdin
```
Comment (Trulsaa) [inline Makefile:2]: Nitpicking: Split on new line for each pipe

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	docker context use default
+
+.PHONY: help
+
+.DEFAULT_GOAL := help
+
+help:
+	@# Help command taken from: https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+	@echo "Usage: make [task]\n"
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```
Comment (Trulsaa) [inline Makefile:28]: Nitpicking: Split on new line for each pipe

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,28 @@
+az-docker-login: ## logs you in to azure and uses those credentials for docker
+	az acr login --name dockercontainerregistry2020 --expose-token|jq .accessToken -r|docker login dockercontainerregistry2020.azurecr.io --username 00000000-0000-0000-0000-000000000000 --password-stdin
+
+build-website: ## Builds website docker image and pushes it to Azure's container registry
+	docker context use default
+	docker build -f website.Dockerfile -t website .
```
Comment (Trulsaa) [inline Makefile:6]: Nitpicking: write flags fully (--tag)

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+	docker tag website dockercontainerregistry2020.azurecr.io/website
+	docker push dockercontainerregistry2020.azurecr.io/website
+
+build-cms: ## Builds cms docker image and pushes it to Azure's container registry
+	docker context use default
+	docker build -f cms.Dockerfile -t cms .
```
Comment (Trulsaa) [inline Makefile:12]: Nitpicking: write flags fully (--tag)

**Review by Trulsaa — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (TheKnarf) [reply Makefile:2]: WONTFIX

File: Makefile (hunk 3 above)
Comment (TheKnarf) [reply Makefile:6]: WONTFIX

File: Makefile (hunk 4 above)
Comment (TheKnarf) [reply Makefile:12]: WONTFIX

File: Makefile (hunk 2 above)
Comment (TheKnarf) [reply Makefile:28]: WONTFIX

**Pull request merged**